### ISSUE TITLE
Stockpile/Workshop hotkeys activate while nicknaming shop

### ISSUE THREAD

**Issue by quinnvoker** [DFHack/dfhack#1267]
Currently using using 0.44.10-alpha1 on 64-bit Linux.
When attempting to nickname a stockpile with the new feature of 44.10, typing any letter tied to a stockpile hotkey (l, M, T, i) would activate the hotkey, even opening new menus, making it impossible to give the stockpile a name that included the letters l or i.

**Comment by lethosor** [comment]
Yeah, we weren't checking for the proper name flag before because it wasn't used there. Good catch!
